README.md

```diff
@@ -3139,6 +3139,44 @@ Other Style Guides
     ];
     ```
 
+  <a name="naming--uppercase"></a>
+  - [23.10](#naming--uppercase) You may optionally uppercase a constant only if it (1) is exported, (2) is a const, and (3) the programmer can trust it (and its nested properties) to never change.
```

Comment:
"is a const" isn't necessarily clear to me; a const in JS is a variable that's never reassigned. Is that what you mean?

Reply:
Literally a `const` and not `let`. The conceptual idea of a constant and JS's implementation of const isn't quite aligned, so this just asserts that in addition to not being reassigned, it _can't_ be reassigned. What is the best way to make this more clear?

Reply:
@ljharb (tagging you since idk if you get a notification otherwise)

Reply:
Gotcha - I'll add backticks around it, but otherwise that's probably good as-is :-)